src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -898,6 +898,7 @@ uct_ib_mlx5_devx_mem_reg(uct_md_h uct_md, void *address, size_t length,
 static ucs_status_t
 uct_ib_devx_dereg_invalidate_rkey_check(uct_ib_mlx5_md_t *md,
                                         uct_ib_mlx5_devx_mem_t *memh,
+                                        struct mlx5dv_devx_obj *dvmr,
```

Comment:
do we still need to pass the rkey?

Reply:
only for log message

Reply:
So it's not relevant anymore since the check is based on dvmr now and not the rkey
let's print the dvmr